docs/source/index.rst

```diff
@@ -108,3 +108,10 @@ When the :py:meth:`.fit() <cleanlab.classification.CleanLearning.fit>` method is
    GitHub <https://github.com/cleanlab/cleanlab>
    PyPI <https://pypi.org/project/cleanlab/>
    Conda <https://anaconda.org/Cleanlab/cleanlab>
+
+.. toctree::
+   :hidden:
+
+   Migrating to v2.0.0 <migrating/migrate_v2>
```

Comment:
> "Migrating to v2.0.0" shows up in the "Links" section on the sidebar. Is that intentional?

@anishathalye Yes that's where I put it, feel free to move it elsewhere if you have a better spot.

Reply:
Seems fine for now, we can consider moving it later.